cuda/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ enable_language( CUDA )
 # Project include(s).
 include( vecmem-compiler-options-cuda )
 
+find_package( CUDAToolkit )
```

Comment:
One more thing... You should add `REQUIRED` here. The configuration should just bomb if CUDA is not available.